api/v1alpha1/search_types.go

```diff
@@ -31,10 +31,10 @@ type Search struct {
 	Status SearchStatus `json:"status,omitempty"`
 }
 
-// SearchSpec defines the desired state of Search
+// SearchSpec defines the desired state of Search.
 type SearchSpec struct {
-	// INSERT ADDITIONAL SPEC FIELDS - desired state of cluster
-	// Important: Run "make" to regenerate code after modifying this file
+	// You can insert additional specification fields.
+	// Important: To regenerate code after you modify the file, run the "make" command.
```

Comment:
Actually, this was a development comment from the initial template. Let's just keep the part about running "make" the top of this file, after the `import(...)` statements.

BTW, this comment also applies to this PR. When you run "make" it will propagate your changes to the *.yaml files that get deployed on the cluster.  @dockerymick if you don't have the correct setup, I can run it and push the changes.